platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/main/java/com/swirlds/demo/consistency/ConsistencyTestingToolMain.java

```diff
@@ -97,21 +86,21 @@ public void run() {
     @Override
     @NonNull
     public ConsistencyTestingToolState newStateRoot() {
-        final ConsistencyTestingToolState state = new ConsistencyTestingToolState();
-        TestingAppStateInitializer.DEFAULT.initConsensusModuleStates(state);
-
+        final ConsistencyTestingToolState state = new ConsistencyTestingToolState(createVirtualMap());
+        TestingAppStateInitializer.initConsensusModuleStates(state);
         return state;
     }
 
     /**
      * {@inheritDoc}
-     * <p>
-     * FUTURE WORK: https://github.com/hiero-ledger/hiero-consensus-node/issues/19002
-     * </p>
      */
     @Override
     public Function<VirtualMap, ConsistencyTestingToolState> stateRootFromVirtualMap() {
-        throw new UnsupportedOperationException();
+        return (virtualMap) -> {
```

Comment:
`newStateRoot()` and `stateRootFromVirtualMap()` can be refactored to share common code

Reply:
Done